### PR TITLE
LRQA-50723 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-50723

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -342,13 +342,62 @@ definition {
...
+
+		User.loginUserPG(
+			password = "test",
+			userEmailAddress = "userea@liferay.com");
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+
+		DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.jpeg,Document_2.docx,Document_3.mp3,Document_4.txt");
+
+		User.logoutAndLoginPG(
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:359]: Can we use the same macro for lines 340 and 351?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 2]
```diff
@@ -342,13 +342,62 @@ definition {
...
+			applications = "Documents and Media",
+			itemsCount = "4");
+
+		LexiconEntry.selectEntry(key_rowEntry = "Document_1.jpeg");
+
+		LexiconEntry.selectEntry(key_rowEntry = "Document_2.docx");
+
+		GDPR.deleteSelectedEntryCP();
+
+		if (IsElementNotPresent(locator1 = "NavTab#ACTIVE_TAB_LINK", key_tab = "Review Data")) {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:382]: See if there is an existing macro that will cover this until line 294

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 3]
```diff
@@ -342,13 +342,62 @@ definition {
 
 	}
 
-	@description = "LPS-77748"
+	@description = "This is a use case for LPS-77748. This test deletes multiple selected entries from a single application."
 	@ignore = "Test Stub"
 	@priority = "3"
 	test CanDeleteMultipleEntriesFromAnApplication {
+		User.logoutPG();
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:349]: Looks like you can use the macro you used in line 359 here

File: portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro [hunk 4]
```diff
@@ -121,6 +121,16 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro deleteSelectedEntryCP {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro:124]: Double check if an existing macro covers this already

File: portal-web/test/functional/com/liferay/portalweb/macros/SignIn.macro [hunk 5]
```diff
@@ -14,6 +14,8 @@ definition {
 			Navigator.openSpecificURL(url = "${baseURL}/web/guest?SM_USER=${userEmailAddress}");
 		}
 
+		User.resetPassword();
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/SignIn.macro:17]: Can you provide an explanation on why we are adding a password reset?

**Comment by patriciajeanperez** [comment]
Looks good! Just a few comments on making sure we check for existing macros to avoid duplicates. I also needed more information on the password reset that was added. Thanks, @epark18!

**Pull request closed without merging**